emanote/src/Emanote/Model/StaticFile.hs

```diff
@@ -74,29 +84,31 @@ readStaticFileInfo fp readFilePath = do
           pure $ Just StaticFileInfoAudio
       | extension == "pdf" ->
           pure $ Just StaticFileInfoPDF
-      | extension `elem` codeExts ->
-          readFilePath fp <&> Just . StaticFileInfoCode
+      | extension `elem` Map.keys codeExts ->
+          readFilePath fp <&> Just . StaticFileInfoCode (codeExts Map.! extension)
```

Comment:
A `Map.lookup` (instead of list lookup via `elem` on `Map.keys`) here would be more efficient right?

Reply:
That's true, I forgot there's `Map.member`

Reply:
Fixed